### ISSUE TITLE
The Portmaster UI is too heavy on low end devices.

### ISSUE THREAD

**Issue by Noir16** [safing/portmaster#1744]
**What would you like to add or change?**:
Lightweight UI and tray notifications that can be implemented for portmaster.

**Why do you and others need this?**:
I understand that the UI has been implemented via Electron for a while now. This seems to be too heavy on startup and shutdown and eats too much memory. and Portmaster's current UI implementation seems to be quite transitional. It appears that they are currently preparing to migrate to an edge webview, but my opinion is that the webview implementation is worse in terms of privacy than weight.

While I think Portmaster does a good job of blocking telemetry from webview, there is no guarantee that everything will work ideally.
So I'm hoping for a lightweight UI and tray notifier with its own implementation.

**Comment by dhaavi** [comment]
Hey @Noir16, thanks for your suggestions.

The UI based on edge webview is already in beta, you can test it by switching to the Beta release channel:
https://wiki.safing.io/en/FAQ/SwitchReleaseChannel

**Comment by Noir16** [comment]
@dhaavi I've already reported on Edge Webview after testing it as well.
Electron and Edge Webview share the commonality of being Chromium-based frameworks, so we didn't see a significant difference in performance.
Also, Electron seems to perform a little better. Edge turns on UI windows faster, but overall starts a little later than Electron.

**Comment by pbzinwindows** [comment]
I agree with this, the UI here is very bad, the new one is mainly causing duplication, it doesn't open, among other defects, unlike other apps
